src/pages/campaignWizardPage.ts

```diff
@@ -0,0 +1,23 @@
+import { Page } from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+
+export default class CampaignWizardPage {
+  private base: PlaywrightWrapper;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+
+  advertiserType = this.page.locator("//label/*[.=' Direct Advertiser ']");
+  companyNameInput = this.page.locator("//input[@name='select_advertiser']");
+  campaignNameInput = this.page.locator("//input[@name='campaign_name']");
+  ymCompany = this.page.locator("//li/*[.='YM Company']");
```

Comment:
Still not sure what this locator is for. The name is confusing.

Reply:
this is the locator for the company that I use to create a campaign. It's name is just "YM Company"

Reply:
What if someone else wanted to create a campaign for their companies? Will they have to create a locator for their companies, too?

Reply:
Yes

Reply:
What if we want to 100 companies, are you going to create 100 locators for 100 companies??